test/bot_test.exs

```diff
@@ -8,32 +10,35 @@ defmodule Slack.BotTest do
   @rtm %{
```

Comment:
It's best to use a fake test web server here and instead of mocks later on. It's relatively safe to assume that unless Slack API misbehaves, the data will arrive as a well formed JSON. The tests will then catch issues with assuming keys to atoms conversion and others.

Mocking results of processing outside data leads to green tests that are likely to lie.